### ISSUE TITLE
Spellchecker LibreOffice

### ISSUE THREAD

**Issue by uxman22222** [OpenMandrivaAssociation/distribution#2785]
**OpenMandriva version**: Openmandriva 4.3 lx 

**Describe the bug**:
Installed myspell-de, but german spellchecker is not available in LibreOffice

**Steps to reproduce**:
install myspell-de, start LibreOffice, try to choose german spellcheck
**Observed behavior**:

**Expected behavior**:

**Additional comment**:
on rock and current the same 

**Logs and screenshots if relevant**

**Comment by berolinux** [comment]
Did you also install libreoffice-l10n-de?

**Comment by uxman22222** [comment]
Yes, it is installed! This one is hard to describe! If there is a myspell or hunspell package installed, usually you have a little indicator (A...) in the field where you choose the default spellchecker in "Tools-Options-Language Settings" in LibreOffice indicating that the spellchecker is installed... I only see this for English, but not for german, what is the default on the system and that user account.

**Comment by uxman22222** [comment]
To test what happens I also installed libreoffice-l10n-fr, myspell-fr and the result is the same....
French spellchecker is also not available in LibreOffice...

**Comment by uxman22222** [comment]
This bug still exists

**Comment by rugyada** [comment]
Did you also install aspell-de ?

**Comment by benbullard79** [comment]
This is a guess but perhaps try going through the stuff posted here: https://ask.libreoffice.org/t/spelling-checking-not-working/37642/3

I would think this problem might be more likely to be answered by LibreOffice folks. I am looking at this but my knowledge on how language software works is weak. Reproducing this  problem on a English (US) system is what I am trying to do.

**Comment by benbullard79** [comment]
Neither German nor French is not in the list of languages here:

![Screenshot_20220524_222459](https://user-images.githubusercontent.com/5704386/170174018-7a321bff-a0c8-4988-ba98-d132834fade1.jpg)

_I believe this is a bug._ Now the question is whether this is an OMLx bug or upstream?

FWIW that list contains Dutch, English, Italian, Norwegian, Polish, Portuguese, Romanian, Russian, Spanish, and Ukrainian. 

I currently have these installed on this system:

```
# rpm -qa | grep libreoffice
libreoffice-help-en_US-7.3.0.3-2.znver1
libreoffice-help-pt_BR-7.3.0.3-2.znver1
libreoffice-l10n-pt_BR-7.3.0.3-2.znver1
libreoffice-style-breeze-7.3.0.3-2.znver1
libreoffice-common-7.3.0.3-2.znver1
libreoffice-draw-7.3.0.3-2.znver1
libreoffice-help-en_GB-7.3.0.3-2.znver1
libreoffice-l10n-en_GB-7.3.0.3-2.znver1
libreoffice-help-es-7.3.0.3-2.znver1
libreoffice-l10n-es-7.3.0.3-2.znver1
libreoffice-help-fr-7.3.0.3-2.znver1
libreoffice-l10n-fr-7.3.0.3-2.znver1
libreoffice-help-it-7.3.0.3-2.znver1
libreoffice-l10n-it-7.3.0.3-2.znver1
libreoffice-help-nb-7.3.0.3-2.znver1
libreoffice-l10n-nb-7.3.0.3-2.znver1
libreoffice-help-nl-7.3.0.3-2.znver1
libreoffice-l10n-nl-7.3.0.3-2.znver1
libreoffice-help-nn-7.3.0.3-2.znver1
libreoffice-l10n-nn-7.3.0.3-2.znver1
libreoffice-help-pl-7.3.0.3-2.znver1
libreoffice-l10n-pl-7.3.0.3-2.znver1
libreoffice-help-ru-7.3.0.3-2.znver1
libreoffice-l10n-ru-7.3.0.3-2.znver1
libreoffice-impress-7.3.0.3-2.znver1
libreoffice-calc-7.3.0.3-2.znver1
libreoffice-kde5-7.3.0.3-2.znver1
libreoffice-l10n-de-7.3.0.3-2.znver1
libreoffice-l10n-ro-7.3.0.3-2.znver1
libreoffice-math-7.3.0.3-2.znver1
libreoffice-writer-7.3.0.3-2.znver1
```
```
# rpm -qa | grep myspell
myspell-de-20211213-1.noarch
myspell-nl_NL-20211213-1.noarch
myspell-it_IT-20211213-1.noarch
myspell-ru_RU-20211213-1.noarch
myspell-pt_BR-20211213-1.noarch
myspell-pl_PL-20211213-1.noarch
myspell-es-20211213-1.noarch
myspell-en-20211213-1.noarch
myspell-fr_BE-20211213-1.noarch
myspell-uk_UA-20211213-1.noarch
myspell-ro-20211213-1.noarch
myspell-no-20211213-1.noarch
myspell-fr_FR-20211213-1.noarch
```
```
# rpm -qa | grep aspell
lib64aspell15-0.60.8-2.znver1
aspell-en-2020.12.07-2.znver1
aspell-0.60.8-2.znver1
```
```
# rpm -qa | grep locales
locales-2.34-6.znver1
locales-fr-2.34-6.znver1
locales-de-2.34-6.znver1
locales-en-2.34-6.znver1
locales-es-2.34-6.znver1
locales-no-2.34-6.znver1
locales-pl-2.34-6.znver1
locales-pt-2.34-6.znver1
locales-ru-2.34-6.znver1
locales-it-2.34-6.znver1
locales-nl-2.34-6.znver1
locales-ro-2.34-6.znver1
locales-uk-2.34-6.znver1
locales-af-2.34-6.znver1
locales-ar-2.34-6.znver1
locales-cs-2.34-6.znver1
locales-da-2.34-6.znver1
locales-he-2.34-6.znver1
locales-hu-2.34-6.znver1
locales-tr-2.34-6.znver1
```

**Comment by benbullard79** [comment]
I am checking this in a up to date ROME znver1 system with:

`libreoffice-common-7.4.1.1-1.znver1`

and of course all the other LO packages. _Looks like you have to install a German spellcheck extension._

![Screenshot_20220923_144006](https://user-images.githubusercontent.com/5704386/192044964-f58dded5-7228-40a3-88a2-49cb6143d6a9.jpg)

![Screenshot_20220923_143737](https://user-images.githubusercontent.com/5704386/192044994-57da1a81-6608-470d-b54b-68991427df11.jpg)

And after installing that it is spell checking the heck out of this English lang. document.

![Screenshot_20220923_144053](https://user-images.githubusercontent.com/5704386/192045083-b036cced-537f-4fc5-9f6c-3119ff7a7472.jpg)

**Comment by benbullard79** [comment]
Ah, but I checked Rock/4.3 znver1 system on hardware and the extension packages are present:

![Screenshot_20220923_151606](https://user-images.githubusercontent.com/5704386/192050381-87716f1b-ae83-4fa9-b29f-6999cdc0b071.jpg)

So this bug report would seem to be invalid. More a question of lack of information, which with LO is very understandable. I did not know about these extensions myself before today.
